Assignment/solution_student/css/style.css

```diff
@@ -27,15 +27,179 @@ html {
 .clearfix:after {
   clear: both;
 }
-/******************************************
-/* BASE STYLES
-/*******************************************/
+
 body {
-  color: black;
-  font-size: 16px;
+  font-size: 13px;
+  font-weight: lighter;
   background: white;
+  padding: 50px;
+  color: #05080C;
+  line-height: 1.5;
+  max-width: 900px;
+  margin: 70px auto;
+  background: #F6F6F6;
+  border-top: 5px solid #222222;
+  font-family: 'open sans',sans-serif;
+  border-bottom: 5px solid #222222;
+  padding-top: none;
+}
+
+header {
+    height: 100px;
+    margin:0 auto;
+    padding:5px;
+    background:#003366;
+    text-align: left center;
+}
+
+.logo {
+	color: white;
+	text-indent: 5px;
+	display: inline;
+}
+
+.nav li{
+	text-align: right center;
+	display: inline-block;
+	float: right;
+	padding: 0;
+	margin: 0;
+	margin-bottom: 20px;
+	list-style: none;
+	width:auto;	
+	color:#f9e42e;
+	font-size: 15px;
+	font-weight: 300px;
+}
+
+.readmore {
+	font-size: 15px;
+	font-weight: bold;
+	float: right;
+	padding-top: 5px;
+	text-decoration: none;
+	margin-bottom: 15px;
+}
+
+.implemented, .traveled {
+	width: 70%;
```

Comment:
Super close with the floating 2 columns, but I think you'd be much better off floating the column itself, and setting the width on the column itself, rather than all of the children elements of it. So take the float and with properties off of this declaration, and instead add them to the `.column1` class as a new declaration:
```
.column1 {
  float: left;
  width: 70%;
}
```
This will also fix your issue with the sidebar where you had to add a negative top margin — you won't have to do that anymore if you do it this way :)